assets/style.styl

```diff
@@ -162,16 +162,6 @@ tour-tooltip
   p
     margin-bottom 15px
 
-  tour-settings
-    display block
-    line-height 1
-    white-space nowrap
-    color #811b34
-    margin-top 15px
-    text-transform uppercase
-    font-size smaller
-    cursor pointer
-
```

Comment:
Not entirely sure how this got into `develop`. It'll be in a future PR for the "don't show hints" feature.